imageio/plugins/pillow.py

```diff
@@ -280,6 +280,11 @@ def write(
         image.
 
         """
+        if "fps" in kwargs:
+            raise NotImplementedError(
```

Comment:
```suggestion
            raise TypeError(
```

Python raises a TypeError for unknown kwargs, and I think we should do the same.